composer.json

```diff
@@ -32,7 +32,8 @@
         "ext-json": "*",
         "guzzlehttp/promises": "^1.4",
         "guzzlehttp/psr7": "^1.7 || ^2.0",
-        "psr/http-client": "^1.0"
+        "psr/http-client": "^1.0",
+        "symfony/deprecation-contracts": "^2.4"
```

Comment:
`^2.2` would be better I think. There are no code changes between 2.2.0 and 2.4.0: https://github.com/symfony/deprecation-contracts/compare/v2.2.0...v2.4.0.

Reply:
Good idea!